frontend/src/components/Utils/time.js

```diff
@@ -78,4 +76,32 @@ export function timezonesForCountry(countryName) {
   return zones.map(e => {
     return e.replace(/[_]/g, ' ');
   });
+}
+
+/**
+ * Get a date in 'YYYY-MM-DD' form. 
+ * 
+ * @param {string} msTimestamp Timestamp, in milliseconds since epoch.
+ * @param {string} timezone The timezone which the string should be returned in.
+ * @returns {string} The date in 'YYYY-MM-DD' form. 
+ */
+export function getDateBarebones(msTimestamp, timezone=null) {
+  if (timezone === null) {
+    return moment.tz(parseFloat(msTimestamp), '').format('YYYY-MM-DD');
```

Comment:
What does an empty second argument do? I could not find the answer on the docs [here](https://momentjs.com/timezone/docs/#/using-timezones/) so could you also let me know where you found this (unless you just found it by poking around)?

Reply:
GMT.